content/install/podman-systemd/axosyslog.service

```diff
@@ -0,0 +1,68 @@
+[Unit]
+Description=AxoSyslog Container
+Wants=NetworkManager.service network-online.target
+After=NetworkManager.service network-online.target
+
+
+[Install]
+WantedBy=multi-user.target default.target
+
+[Container]
+# Sets User and Group ID of container syslog-ng process; should match UID/GID of desired host user
+# User= and Group= values must be numeric; this is a hard requirement with strict input validation
+# Example: host passwd entry syslogng:x:1003:1004 => User=1003 Group=1004
+# Any symbolic representation, environment variable, or other non-numeric value will
+# be ignored and the container run as root (UID 0).
+User=1003
+Group=1004
+
+
+ContainerName=AxoSyslog
+
+
+AddCapability=CAP_NET_BIND_SERVICE CAP_CHOWN CAP_FOWNER CAP_DAC_OVERRIDE CAP_DAC_READ_SEARCH CAP_SYS_ADMIN
+
+
+Image=${AXOSYSLOG_IMAGE}
+
+
+Volume=${PERSIST_MOUNT}:/var/lib/syslog-ng:z
+Volume=${CONFIG_MOUNT}:/etc/syslog-ng:z
+Volume=${DISKBUF_MOUNT}:/opt/dskbuf:z
```

Comment:
let's use /opt/axosyslog for the mount points.

/opt/axosyslog/etc
/opt/axosyslog/var/lib

Reply:
let's drop DISKBUF_MOUNT